### PR TITLE
WIP: enable OSX support

### PR DESCRIPTION
**Pull request by stevewallin**
removed additional parenthesis in rconsole.c to allow compilation on osx

Signed-off-by: Steve Wallin <stevewallin@gmail.com>

### CODE REVIEW

File: runtime/exelib/common/rconsole.c [hunk 1]
```diff
@@ -60,7 +60,7 @@ remoteConsole_parseCmdLine(J9PortLibrary *portLibrary, UDATA lastLegalArg, char
 	port = NULL;
 	filepath = NULL;
 	for (i = 1; i <= lastLegalArg; i++) {
-		if ((argv[i][0] == '-')) {
+		if (argv[i][0] == '-') {
```
Comment (keithc-ca) [inline runtime/exelib/common/rconsole.c]: Can you swap the arguments so the constant is on the left?

Comment (stevewallin) [reply runtime/exelib/common/rconsole.c]: did you want me to go through the rest of the function / file or just submit the swap for this one line for now ?

Comment (keithc-ca) [reply runtime/exelib/common/rconsole.c]: Actually, I think that sort of 'cleanup' should be done in a separate PR.
You will need to sign the Eclipse Contributor Agreement (ECA).

Comment (stevewallin) [reply runtime/exelib/common/rconsole.c]: I have a ECA signed yesterday so hopefully all in the system by now

Comment (charliegracie) [reply runtime/exelib/common/rconsole.c]: It is weird.  You did not fail the IP-Validation.  It says that the service is down.

Comment (keithc-ca) [reply runtime/exelib/common/rconsole.c]: The format of that line (in your commits) should be:
```
Signed-off-by: Steve Wallin <steve_wallin@uk.ibm.com>
```

**Comment by stevewallin** [comment]
attempting to get the ip-validation to re-run
Signed-off-by: Steve Wallin <steve_wallin@uk.ibm.com>

**Comment by DanHeidinga** [comment]
@stevewallin The git commit message is missing the `Signed-off-by: Steve Wallin steve_wallin@uk.ibm.com` line.  You'll need to amend the commit and push it again.

Something like:
`$ git commit --amend`
Add the signed off by line to the end of the commit message
`$ git push -f origin patch-1`

You'll need to force push due to the history rewrite that amending a commit causes.

**Comment by stevewallin** [comment]
closing and creating new pr due to issues with ip-validation

**Pull request closed without merging**